### PR TITLE
[8.0][MIG] backport mail template inactive

### PR DESCRIPTION
**Pull request by thomaspaulb**

### CODE REVIEW

File: mail_template_inactive/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+{
+    'name': 'Email Template Inactive',
+    'summary': 'Add active flag to Email Templates',
+    'author': 'Noviat',
+    'category': 'Mail',
+    'website': 'https://www.noviat.com',
+    'version': '8.0.1.0.0',
+    'license': 'AGPL-3',
+    'depends': [
+        'mail',
```
Comment (thomaspaulb) [inline mail_template_inactive/__openerp__.py]: `email_template`

**Review by thomaspaulb — COMMENTED** [review]
@Kiplangatdan

**Comment by luc-demeyer** [comment]
this PR looks ok to me (based upon quick code review, no testing done yet), 
Any reason why it's still marked WIP: ?

**Comment by thomaspaulb** [comment]
We hadn't finished the internal review, but now we have, so have removed the WIP

**Pull request merged**